finance_dl/amazon.py

```diff
@@ -273,14 +277,18 @@ def retrieve_invoices(self, invoice_hrefs):
             # Wait until it is all generated.
             def get_source():
                 source = self.driver.page_source
-                if 'Grand Total:' in source:
+                if 'Grand Total:' in source or 'Order Canceled' in source:
```

Comment:
a bit late for the party, but @jktomer can you explain which case is handled by `Order Canceled`? well, probably order canceled, but are there cases where the source does not contain a grand total?

Background: I ported the amazon downloader to german language in #50 and would like to cover the appropriate translation (if applicable). The amazon websites tend to differ quite a bit, so maybe this case doesn't even exist on .de ...

Thanks for your work on the downloader!